recipes-extended/xen/files/hvmloader-legacy-seabios-optionroms.patch

```diff
@@ -43,9 +43,9 @@ PATCHES
 +#SUBDIRS-$(CONFIG_SEABIOS) += seabios-dir
```

Comment:
Since we run configure with '--with-system-seabios', CONFIG_SEABIOS is set to 'n' in `config/Tools.mk` and seabios-dir won't be included in SUBDIRS.  So this hunk could be dropped.